Source/WebKit/UIProcess/Inspector/mac/WebInspectorUIProxyMac.mm

```diff
@@ -476,6 +476,9 @@ - (IBAction)_popUpButtonAction:(id)sender
     m_inspectorWindow = WebInspectorUIProxy::createFrontendWindow(savedWindowFrame, InspectionTargetType::Local);
     [m_inspectorWindow setDelegate:m_objCAdapter.get()];
 
+    if (inspectedPage())
+        [m_inspectorWindow setInspectedWebView:inspectedPage()->cocoaView().get()];
+
```

Comment:
Should we put this inside `WebInspectorUIProxy::createFrontendWindow` instead so we do it at the same time we set `forRemoteTarget` along with setting up the rest of the window itself? It should effectively be a no-op for the `RemoteWebInspectorUIProxyMac` since we won't have an inspectedPage.

Reply:
Sure.